### ISSUE TITLE
[Feature Request] Partial rendering to handle big JSON docs

### ISSUE THREAD

**Issue by stayingalivee** [AykutSarac/jsoncrack.com#234]
### Feature

I tried to load a 50k line JSON and it failed to render the big graph.

One can incrementally render the graph Breadth-First with all children nodes collapsed 
children nodes can be rendered once the parent is clicked, this should be enough to handle big files since no one will ever navigate > 8000 nodes graph.

**Comment by AykutSarac** [comment]
Good idea, @VladCuciureanu recently submitted a PR that collapses/expands all nodes. An option into settings to collapse all nodes by default could be added in matter of performance improvement.
